### PR TITLE
Update Docs to deprecate the exportPublic  feature

### PR DESCRIPTION
**Pull request by prb112**
Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

**Review by lmsurpre — APPROVED** [review]
LGTM

File: docs/src/pages/guides/FHIRServerUsersGuide.md [hunk 1]
```diff
@@ -1408,7 +1408,7 @@ Note, the deletion of an a job is split into two phases, ACCEPTED (202) response
 
 By default, the exported `ndjson` file is configured with public access automatically and with 2 hours expiration time, the randomly generated secret in the path is used to protect the file. Please note that IBM COS does not support expiration time for each single COS object, so please configure retention policy (e.g, 1 day) for the bucket if IBM COS is used. For both Amazon S3 and IBM COS, please remember that public access should never be configured to the bucket itself.
 
-Note: `fhirServer/bulkdata/storageProviders/(source)/exportPublic` can be set to "false" to disable public access. Also, *minio* doesn't support object level ACL, so access token is always needed to download the exported `ndjson` files.
+As of Version 4.8.1, the exportPublic is deprecated and will be removed in future versions. `fhirServer/bulkdata/storageProviders/(source)/exportPublic` can be set to "false" to disable public access. Also, *minio* doesn't support object level ACL, so access token is always needed to download the exported `ndjson` files.
```
Comment (lmsurpre) [inline docs/src/pages/guides/FHIRServerUsersGuide.md:1411]: we could probably go one step further and flip the default to `false`, but possibly  the doc update alone is sufficient to convey that "you should not rely on exportPublic=true going forward".

Comment (prb112) [reply docs/src/pages/guides/FHIRServerUsersGuide.md:1411]: Let me know if you want me to flip the default, I can do that.

Comment (prb112) [reply docs/src/pages/guides/FHIRServerUsersGuide.md:1411]:
```
    @Override
    public boolean isStorageProviderExportPublic(String provider) {
        return FHIRConfigHelper.getBooleanProperty("fhirServer/bulkdata/storageProviders/" + provider + "/exportPublic", Boolean.FALSE);
    }

Comment (prb112) [reply docs/src/pages/guides/FHIRServerUsersGuide.md:1411]: It's already false. :)

File: docs/src/pages/guides/FHIRServerUsersGuide.md [hunk 2]
```diff
@@ -1408,7 +1408,7 @@ Note, the deletion of an a job is split into two phases, ACCEPTED (202) response
 
 By default, the exported `ndjson` file is configured with public access automatically and with 2 hours expiration time, the randomly generated secret in the path is used to protect the file. Please note that IBM COS does not support expiration time for each single COS object, so please configure retention policy (e.g, 1 day) for the bucket if IBM COS is used. For both Amazon S3 and IBM COS, please remember that public access should never be configured to the bucket itself.
```
Comment (lmsurpre) [inline docs/src/pages/guides/FHIRServerUsersGuide.md:1409]:
```suggestion
Please note that IBM COS does not support retention limits for individual COS objects, so please configure an appropriate retention policy (e.g. 1 day) at the bucket level.
```

**Pull request merged**